olp-spec.md

```diff
@@ -41,8 +43,9 @@ Here's a very simple `outlearn.json` file:
   "paths" : [
     {
       "name" : "learning-to-tango",
+      "freshnessDate" : 2015-05-25,
       "title" : "Learning to Dance the Tango",
-      "description" : "A learning path for those new to the marvelous tango dance, we'll go through all the basics, and all that's left for you to do is practice until your feet are sore!",
+      "description" : "A learning path for those new to the marvelous tango dance, we'll go through all the basics.",
```

Comment:
@wkoffel The layout allows about 130 characters before it does not all show up so I shortened this.